src/index.html

```diff
@@ -169,19 +169,19 @@ <h1 class="header__title">
           </nav>
 
           <div class="menu__contact">
-            <div class="menu__contact--info">
+            <div class="menu__phone">
               <div class="menu__phone--title">The Met Fifth Avenue</div>
               <a
-                class="menu__phone--call"
+                class="menu__phone--call-number"
                 href="tel:+1 212-535-7710"
               >
                 +1 212-535-7710
               </a>
             </div>
-            <div class="menu__contact--info">
+            <div class="menu__phone--info">
```

Comment:
fixed this mistake on local already. don't want commit just for it. It doesn't affect how the site works.